src/components/Doctors/doctor.js

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import axios from 'axios';
+import { getADoctor } from '../../store/doctors/doctors';
+import styles from './doctors.module.css';
+
+const url = 'https://pick-a-doc.herokuapp.com/api/';
+
+const GetDoctor = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const doctor = useSelector((state) => state.doctorReducer);
+  const currentUser = useSelector((state) => state.usersReducer.currentUser);
+  const { headers } = currentUser;
+  const [state, setState] = React.useState({
+    doctor_id: '',
+    user_id: '',
+    date: '',
+    time: '',
+    notes: '',
+  });
+
+  const submitForm = (evt) => {
+    evt.preventDefault();
+    const appointment = {
+      doctor_id: doctor.doctor.id,
+      user_id: currentUser.attributes.id,
+      date: `${state.date} ${state.time}`,
+      notes: state.notes,
+    };
+    console.log(appointment);
+    console.log(currentUser.headers);
+    axios
+      .post(`${url}appointments`, appointment, { headers })
+      .then(() => {
+        navigate('/appointments');
+      })
+      .catch((error) => {
+        console.error('There was an error!', error);
```

Comment:
- [ ] Remove `console.log` and `console.error`, since it's not a good practice. I suggest that you use a way of showing a message to the user or send it somewhere else to debug (which I believe it's not the case in this capstone project).

Reply:
I will remove the console.log, but because of the time of the presentation, I will keep the console. error until we handle it to show to the user after then.